code401challenges/src/test/java/code401challenges/utilities/MultiBracketValidationTest.java

```diff
@@ -1,4 +1,29 @@
 package code401challenges.utilities;
 
+import org.junit.Test;
+
+import static code401challenges.utilities.MultiBracketValidation.multiBracketValidation;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
 public class MultiBracketValidationTest {
+// HAPPY TEST
+    @Test public void allCorrectOrderTest(){
+        String input = "{}{Code}[Fellows](())";
+        boolean actual = multiBracketValidation(input);
+        assertTrue(actual);
+    }
```

Comment:
I would get rid of the variable you're using to store the boolean results from the method call and just do that in the assert statement. It would look like this: `assertTrue(multiBracketValidation(input));`